lib/PuppeteerSharp.Tests/PageTests/EmulateMediaFeaturesTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Threading.Tasks;
+using PuppeteerSharp.Media;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PuppeteerSharp.Tests.PageTests
+{
+    [Collection(TestConstants.TestFixtureCollectionName)]
+    public class EmulateMediaFeaturesAsyncTests : PuppeteerPageBaseTest
+    {
+        public EmulateMediaFeaturesAsyncTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ShouldWork()
```

Comment:
Usually I try to not have so much code that is so similar, I'm sure this could be reduced a lot using something like NUnit `ValuesAttribute`, because it's really hard to maintain.
But on the other hand, this will probably never change, so it's ok I guess ;)